index.js

```diff
@@ -0,0 +1,24 @@
+const menuBtn = document.querySelector('.menu-btn');
+const headerNavMobile = document.querySelector('.header-nav-mobile');
+const cancelCross = document.querySelector('.cancel-cross');
+const linkPortfolio = document.querySelectorAll('.menu-link');
+//const linkAbout = document.querySelector('.menu-link');
+//const linkContact = document.querySelector('.menu-link');
+
+menuBtn.addEventListener('click', () => {
+    menuBtn.classList.add('hidden')
+    headerNavMobile.classList.remove('hidden')
+});
+
+cancelCross.addEventListener('click', () => {
+    menuBtn.classList.remove('hidden')
+    headerNavMobile.classList.add('hidden')
+});
+
+linkPortfolio.addEventListener('click', () => {
+    headerNavMobile.classList.add('hidden');
+    menuBtn.classList.remove('hidden');
+  });
```

Comment:
**OPTIONAL** - According to the project requirements, when the user clicks on a navigation link, the menu should close, this isn't so in your project, please fix this 🙏